akka-actor/src/main/resources/reference.conf

```diff
@@ -25,6 +25,17 @@ akka {
   # (akka.actor.ActorSystem.Settings, akka.event.EventStream) parameters.
   logging-filter = "akka.event.DefaultLoggingFilter"
 
+  # Specifies the default loggers dispatcher
+  loggers-dispatcher = "akka.loggers.pinned-dispatcher"
```

Comment:
@patriknw it is arguable, if most users complain that they don't want more threads they can change that but the remaining population what do you think will do? I know I will prefer a dedicated blocking I/O thread, but that's just me, what do you think is the general consensus?